packages/operator/docs/swagger.yaml

```
@@ -422,6 +425,14 @@ definitions:
     type: object
   ModelRoute:
     properties:
+      createdAt:
+        description: CreatedAt
+        type: string
+      default:
+        description: |-
+          Default routes cannot be deleted by  They are managed by system
+          One ModelDeployment as exactly one default route that give 100% traffic to the model
```

```suggestion
          Default routes cannot be deleted by user. They are managed by system
          One ModelDeployment has exactly one default Route that gives 100% traffic to the model
```
I suppose it should be in this way: https://github.com/odahu/odahu-flow/pull/439/files?file-filters%5B%5D=.go&file-filters%5B%5D=.json&file-filters%5B%5D=.mod&file-filters%5B%5D=.py&file-filters%5B%5D=.robot&file-filters%5B%5D=.sql&file-filters%5B%5D=.yaml&hide-deleted-files=true#diff-3ea98d649ec32d490f257bdd6fd340869105d6be75c6855f5c1696bbd8e08cedR27